CHANGELOG.md

```diff
@@ -10,7 +10,8 @@
 For information on changes in released versions of Teku, see the [releases page](https://github.com/ConsenSys/teku/releases).
 
 ## Unreleased Changes
-
+ - Validator Registration signature integration with external signer.
+ - 
```

Comment:
This should go under the `Additions and Improvements` section sub-category below (all the changes in this file are unreleased).

Reply:
👍🏼 done.